packages/requisitions/src/RequestRequisition/DetailView/Toolbar/Toolbar.tsx

```diff
@@ -1,27 +1,43 @@
-import React, { FC } from 'react';
+import React, { Dispatch, SetStateAction } from 'react';
 import {
   AppBarContentPortal,
   InputWithLabelRow,
   BufferedTextInput,
   Grid,
   useTranslation,
+  SearchBar,
 } from '@openmsupply-client/common';
 import { InternalSupplierSearchInput } from '@openmsupply-client/system';
 import { useRequestFields, useIsRequestDisabled } from '../../api';
 import { ToolbarDropDown } from './ToolbarDropDown';
 import { ToolbarActions } from './ToolbarActions';
 
-export const Toolbar: FC = () => {
-  const t = useTranslation('replenishment');
+interface ToolbarProps {
+  filter: {
+    itemFilter: string;
+    setItemFilter: Dispatch<SetStateAction<string>>;
+  };
+}
 
+export const Toolbar = ({
+  filter: { itemFilter, setItemFilter },
+}: ToolbarProps) => {
+  const t = useTranslation('replenishment');
   const isDisabled = useIsRequestDisabled();
   const { theirReference, update, otherParty } = useRequestFields([
     'theirReference',
     'otherParty',
   ]);
 
   return (
-    <AppBarContentPortal sx={{ display: 'flex', flex: 1, marginBottom: 1 }}>
+    <AppBarContentPortal
+      sx={{
+        display: 'flex',
+        flex: 1,
+        marginBottom: 1,
+        flexDirection: 'column',
```

Comment:
Made outermost flex box `column` and placed the Search and Dropbdown components in their own `<Grid>` outside the one the other components are in -- otherwise Search box was pulling the `<ToolbarActions>` stuff over to the right and it would move as the SearchBar expanded -- not pretty.

Reply:
Originally I didn't like the RH layout of the item filter, I looked at 'Internal Orders' first and thought on that screen it would be better on the left..
...but then saw response reqs, and it is better on the right there. And consistency is important! and on internal orders, I like having all the fields related to the requisition together and all the 'view' elements ( filters and such) together.

so.. 👍  and 😁

Reply:
Yeah, I went through the exact same thought process 😄